src/encoded/types/dataset.py

```diff
@@ -865,7 +865,45 @@ class Series(Dataset, CalculatedSeriesAssay, CalculatedSeriesAssayType, Calculat
                 'pipeline_version',
                 'title',
             ],
-        )
+        ),
+        Path('series_files', include=[
+            '@id',
+            '@type',
+            'title',
+            'file_type',
+            'file_format',
+            'file_format_type',
+            'output_type',
+            'assembly',
+            'biological_replicates',
+            'technical_replicates',
+            'read_length',
+            'mapped_read_length',
+            'run_type',
+            'paired_end',
+            'paired_with',
+            'index_of',
+            'derived_from',
+            'file_size',
+            'lab',
+            'md5sum',
+            'dbxrefs',
+            'href',
+            'genome_annotation',
+            'platform',
+            'controlled_by',
+            'status',
+            'no_file_available',
+            'restricted',
+            's3_uri',
+            'analyses',
+            'preferred_default',
+            'processed',
+            'output_category',
+        ]),
+        Path('series_files.lab', include=['title']),
+        Path('series_files.platform', include=['title']),
+        Path('series_files.analyses', include=['title', 'status']),
```

Comment:
Probably a good idea to include `@id` and `@type` for these too. Otherwise difficult to tell what exactly is embedded.